src/MIDebugPackage/MIDebugPackage.csproj

```diff
@@ -8,12 +8,17 @@
     <!--Visual Studio will prompt to upgrade the project unless $(MinimumVisualStudioVersion)==$(VisualStudioVersion), to make
     VS happy, we will conditionally set MinimumVisualStudioVersion based on what version is opening the project -->
     <MinimumVisualStudioVersion Condition="'$(VisualStudioVersion)' == '15.0'">15.0</MinimumVisualStudioVersion>
-    <MinimumVisualStudioVersion Condition="'$(VisualStudioVersion)' != '15.0'">14.0</MinimumVisualStudioVersion>
+    <MinimumVisualStudioVersion Condition="'$(VisualStudioVersion)' != '15.0'">16.0</MinimumVisualStudioVersion>
```

Comment:
change back to 14.0

Reply:
You can use this to make sure VS doesn't put up the bogus upgrade prompts --

```
    <!--Visual Studio will prompt to upgrade the project unless $(MinimumVisualStudioVersion)==$(VisualStudioVersion), to make
    VS happy, we will conditionally set MinimumVisualStudioVersion based on what version is opening the project -->
    <VisualStudioVersion Condition="'$(VisualStudioVersion)' == ''">14.0</VisualStudioVersion>
    <MinimumVisualStudioVersion>$(VisualStudioVersion)</MinimumVisualStudioVersion>
```



---
In reply to: [252509323](https://github.com/Microsoft/MIEngine/pull/831#discussion_r252509323) [](ancestors = 252509323)